cocotb/share/makefiles/Makefile.inc

```
@@ -75,10 +81,17 @@ ifeq ($(SIM_DEFINE),XCELIUM)
 SIM_DEFINE = IUS
 endif
 
-export LIB_DIR=$(COCOTB_PY_DIR)/cocotb/libs
-
 ifeq ($(OS),Msys)
 LIB_EXT    := dll
 else
 LIB_EXT    := so
 endif
+
+ifeq ($(OS),Msys)
+    # On Windows there is no support for @rpath/LD_LIBRARY_PATH so we need to set PATH to find cocotb libraries
+    # https://docs.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order?redirectedfrom=MSDN#standard-search-order-for-desktop-applications
+    export PATH := $(LIB_DIR):$(PATH)
+
+    # For tcl scripts the path to dll has to be in form of C:/User/..
+    export LIB_DIR := $(shell cygpath -m '$(LIB_DIR)')
```

```suggestion
    LIB_DIR := $(shell cygpath -m '$(LIB_DIR)')
```
This isn't exported on other platforms, is that intentional?